src/core/lib/iomgr/work_serializer.cc

```diff
@@ -24,6 +24,8 @@ namespace grpc_core {
 
 DebugOnlyTraceFlag grpc_work_serializer_trace(false, "work_serializer");
 
+namespace {
+
 struct CallbackWrapper {
```

Comment:
While you're at it, this can also be private within `WorkSerializerImpl`.

Reply:
I think it's better to encapsulate implementation details inside of the class that they're used in.  If we ever add another class to this file, that class will not need to see this code.

I generally use an anonymous namespace if the class declaration is in a .h file, since I prefer not to have to publish implementation details in the .h file if I can avoid it.  But if the class declaration is already in the .cc file, private methods seem better.

Reply:
https://abseil.io/tips/186 (go/totw/186) prefers functions in an anonymous namespace for the following reasons - 
```
Benefits over private methods include:

Inputs and outputs are clear since they are (much more likely to be) specified via arguments or the return value. Note that a method may read any member variable and a non-const method may modify any non-const member. Conversely, a non-member function may only read or modify according to its interface (except for globals).
The class API is simpler and shorter, and hence easier to read—unnecessary private methods may make it difficult to find inheritance-related private declarations or declarations after the class.
```